src/components/common/ButtonActionSelector.tsx

```diff
@@ -138,7 +138,7 @@ const PurchaseProductSelector: React.FC<{
                 title={<ProductTypeLabel productType={productType as PartBriefProductType} />}
                 checkable={false}
               >
-                {briefProducts[productType as ProductType]?.map(product => (
+                {briefProducts[productType as Exclude<ProductType, 'Program'>]?.map(product => (
```

Comment:
replace `Object.keys` with utility function `keys` from ramda
then you will not need `Type Assertion`

Reply:
the suggestion should be referred to line 133?

Reply:
can refer to [here](https://github.com/urfit-tech/lodestar-app-admin/pull/112/files#diff-d26dc61b2d00aae91dcd326d84f82096e15b79edb2c446519975d6f8cfa60c41)